server/controllers/chatControllers.js

```diff
@@ -0,0 +1,189 @@
+const asyncHandler = require("express-async-handler");
+const Chat = require("../models/chatModel");
+const User = require("../models/userModel");
+
+//Create or fetch One to One Chat
+const accessChat = asyncHandler(async (req, res) => {
+  const { userId } = req.body;
+
+  if (!userId) {
+    console.log("UserId param not sent with request");
+    return res.sendStatus(400);
+  }
+
+  var isChat = await Chat.find({
```

Comment:
Is "var" necessary? Should be let or const, let seems the appropriate scope

Reply:
Ugh I am the worst